src/main/java/redis/clients/jedis/ConnectionFactory.java

```diff
@@ -17,21 +17,27 @@ public class ConnectionFactory implements PooledObjectFactory<Connection> {
   private static final Logger logger = LoggerFactory.getLogger(ConnectionFactory.class);
 
   private final JedisSocketFactory jedisSocketFactory;
-
   private final JedisClientConfig clientConfig;
+  private ClientSideCache clientSideCache = null;
```

Comment:
You used cscache elsewhere (Connection.java), let's aim for consistency throughout.